pages/faqs/faq-inspector.mdx

```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
 ### What kind of alerts can I set, if any?
 
 You can receive email alerts when new issues are detected. Head to the Issues dashboard on avo.app to configure email alerts.
+
+### When will the "bubble" from the In-App Inspector (aka Avo Debuggers) be visible in my app?
+
+The "bubble" of the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers) is visible when [the debugger is initialized](https://github.com/avohq/react-native-analytics-debugger) in a `dev` environment. Read more about the 
```

Comment:
I think the answer to this question really depends on the platform you are using. We're linking to the standalone react-native debugger there, but afaik the visual debugger has also been embedded into all the Inspector SDKs.

The visual debugger is also made visible on web without any additional dependencies on web if you're running Avo Functions in dev mode.

@alex-tpom6oh could you summarize how you can get the In-App inspector up and running on each platform? Maybe those details can live in this faq, or we could link to more details somewhere else in the docs?

Reply:
There are 2 approaches in bundling the debugger currently.
The old one - when it is bundled with Avo functions and the new one - when it is bundled with Inspector.

The new one is currently available in Android and iOS. In this case the event path is `Avo Function | manual tracking -> Inspector -> Debugger`:
1. iOS - visible if Inspector SDK is included in the project and Inspector is initialized in dev or staging mode. (there is a method to show/hide the visual inspector, can use it in any environment). 
2. Android - visible if the `android-avo-inspector:dev:` gradle dependency is included in the project and Inspector is initialised in dev or staging mode. (there is a method to show/hide the visual inspector, can use it in any environment)
The `android-avo-inspector:prod:` gradle dependency does not have the visual inspector included.


The old one is on Web and React Native. In this case the event path is `Avo Function -> Debugger`:
1. Web - visible on without any additional dependencies on web if you're running Avo Functions in dev mode.
2. React Native - not shown automatically. You need to include `react-native-analytics-debugger` dependency, create instance of the debugger, pass it to initAvo and then call `AvoDebugger.showDebugger({mode: 'bar' // or 'bubble'})` as described here https://github.com/avohq/react-native-analytics-debugger

(Android and iOS also have this mechanism, but I think we should deprecate it and not mention in favour of the Inspector bundle)

Reply:
Not sure how deep into the details should we go here?